ChallengesWithTestsMarkVII/ChallengesSet01.cs

```diff
@@ -8,37 +8,75 @@ public class ChallengesSet01
     {
         public bool AreTwoNumbersTheSame(int num1, int num2)
         {
-            throw new NotImplementedException();
+            return num1 == num2;
         }
 
         public double Subtract(double minuend, double subtrahend)
         {
-            throw new NotImplementedException();
+            double result = (minuend - subtrahend);
+            return result;
         }
 
         public int Add(int number1, int number2)
         {
-            throw new NotImplementedException();
+            return (number1 + number2);
```

Comment:
The parentheses are unnecessary, but can improve readability.  For simple lines like this one, though, they could be removed